### PR TITLE
[DI] Document the container preload tags

### LINKED ISSUE #13494: [DI] add tags `container.preload`/`.no_preload` to declare extra classe…
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#36195
| PR author(s) | @nicolas-grekas
| Merged in    | 5.1-dev

### PR DESCRIPTION
**Pull request by javiereguiluz**
Fixes #13494.

### CODE REVIEW

File: performance.rst [hunk 1]
```diff
@@ -122,6 +122,10 @@ The preload file path is the same as the compiled service container but with the
     ; php.ini
     opcache.preload=/path/to/project/var/cache/prod/srcApp_KernelProdContainer.preload.php
 
+Use the :ref:`container.no_preload <dic-tags-container-nopreload>` and
+:ref:`container.preload <dic-tags-container-preload>` service tags to define
+which classes should or should not be preloaded PHP.
```
Comment (noniagriconomie) [inline performance.rst:127]: As you put first the no preload, maybe better to write should not or should (in same order) wdyt?

File: reference/dic_tags.rst [hunk 2]
```diff
@@ -14,6 +14,8 @@ Tag Name                                  Usage
 `auto_alias`_                             Define aliases based on the value of container parameters
 `console.command`_                        Add a command
 `container.hot_path`_                     Add to list of always needed services
+`container.no_preload`_                   Remove a class from the list of classes to preload by PHP
```
Comment (noniagriconomie) [inline reference/dic_tags.rst:17]: to not preload?

File: reference/dic_tags.rst [hunk 3]
```diff
@@ -212,6 +214,109 @@ for services and their class hierarchy. The result is as significant performance
...
+--------------------
+
+**Purpose**: Remove a class from the list of classes to preload by PHP
+
+.. versionadded:: 5.1
+
+    The ``container.no_preload`` tag was introduced in Symfony 5.1.
+
+When using `PHP class preloading`_, this tag allows you to define that some
+class should not be preloaded by PHP. Add this tag to any service and its
```
Comment (noniagriconomie) [inline reference/dic_tags.rst:229]: some classes (plural no?)

File: reference/dic_tags.rst [hunk 4]
```diff
@@ -212,6 +214,109 @@ for services and their class hierarchy. The result is as significant performance
...
+
+**Purpose**: Add some class to the list of classes to preload by PHP
+
+.. versionadded:: 5.1
+
+    The ``container.preload`` tag was introduced in Symfony 5.1.
+
+When using `PHP class preloading`_, this tag allows you to define which PHP
+classes should be preloaded. This can improve performance by making some of the
+classes used by your service always available for all requests:
```
Comment (noniagriconomie) [inline reference/dic_tags.rst:277]: I would add « between server restart » or something like that for the hint :)

**Review by noniagriconomie — COMMENTED** [review]
Nice feature and doc additions

File: reference/dic_tags.rst (hunk 2 above)
Comment (javiereguiluz) [reply reference/dic_tags.rst:17]: I've reworded as "Remove a class from the list of classes preloaded by PHP"

**Comment by javiereguiluz** [comment]
@noniagriconomie thanks for your review. I've made most of the changes.

**Pull request merged**